bin/metrics-per-process.py

```diff
@@ -89,6 +90,21 @@
 'CLOSING',
 'NONE'
 ]
+HANDLER_STATS = {
+  'cpu.user': lambda ctx: ctx.cpu_times().user,
+  'cpu.system': lambda ctx: ctx.cpu_times().system,
+  'cpu.percent': lambda ctx: ctx.cpu_percent(),
+  'threads': lambda ctx: ctx.num_threads(),
+  'memory.percent': lambda ctx: ctx.memory_percent(),
+  'fds': lambda ctx: ctx.num_fds(),
+  'ctx_switches.voluntary': lambda ctx: ctx.num_ctx_switches().voluntary,
+  'ctx_switches.involuntary': lambda ctx: ctx.num_ctx_switches().involuntary,
+  'io_counters.read_count': lambda ctx: ctx.io_counters().read_count,
+  'io_counters.write_count': lambda ctx: ctx.io_counters().write_count,
+  'io_counters.read_bytes': lambda ctx: ctx.io_counters().read_bytes,
+  'io_counters.write_bytes': lambda ctx: ctx.io_counters().write_bytes,
+}
+MEMORY_STATS = ['rss', 'vms', 'shared', 'text', 'lib', 'data', 'dirty']
```

Comment:
you should freeze constants.

Reply:
you mean use a tuple instead of a list?

Reply:
disregard I was clearly too tired when I reviewed this.